src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -99,25 +99,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $this->validateInput($input);
+        $object = $this->getContainerBuilder();
```

Comment:
I think the reason behind naming it `$object` is that it could either be an instance of ContainerBuilder or the ParameterBag.